Makefile.in

```diff
@@ -29,10 +29,10 @@ ES_IMAGE ?= docker.elastic.co/elasticsearch/elasticsearch-oss
 ES_IMAGE_TAG ?= 7.10.2
 
 SW_OAP_IMAGE ?= ghcr.io/apache/skywalking/oap
-SW_OAP_IMAGE_TAG ?= 8b2fe884152d7bb03e0a8ac0753fdc84df541ce9
+SW_OAP_IMAGE_TAG ?= 8a82ac885d29e0fa4df9c3923edd0dc5830cad2c
```

Comment:
https://github.com/apache/skywalking/actions/runs/5621499911

You didn't check, this commit doesn't push the image successfully, I just rerun it.
Let's see.

Reply:
Please retry, the images are ready now.